pkgs/development/libraries/libevent/default.nix

```diff
@@ -1,4 +1,9 @@
-{ stdenv, fetchurl, openssl, findutils }:
+{
+  stdenv, fetchurl, findutils,
```

Comment:
This formatting is a bit unusual. Typically the comma is at the beginning of the line. Something like
```
{ stdenv, fetchurl, findutils
, enableOpenssl ? true, openssl
}

Reply:
I've seen the comma go both at the beginning and the end and I also find it a bit unbalanced to have a bunch of items on the first line with the open bracket but none on the last line with the close bracket. That being said, it's a minor quibble so I've changed to your recommendation here.